packages/ui5-task-zipper/test/zipper.test.js

```diff
@@ -11,9 +11,10 @@ const test = require("ava");
  *
  * @param {string} zip path to a zip file
  * @param {string} needle path/file to find in the zip
- * @returns {Promise<true|false>} true when needle is found in zip, false otherwise
+ * @param {boolean} readFileContent whether to read the file content
+ * @returns {Promise<true|string|false>} true when needle is found in zip, string if file content could be read, false otherwise
  */
-async function promisifiedNeedleInHaystack(zip, needle) {
+async function promisifiedNeedleInHaystack(zip, needle, readFileContent = false) {
```

Comment:
Setting the default to `false` is optional as normally the unset parameter is `undefined` which is a falsy value

Reply:
Correct, I personally like setting defaults as it improves readability.